### ISSUE TITLE
Single point in signal not plotted

### ISSUE THREAD

**Issue by Fruchtzwerg94** [ScottPlot/ScottPlot#1949]
Using a signal with only one datapoint will not show up this single point:
```
WpfPlot1.Plot.AddSignal(new double[] { 1 }, 1, System.Drawing.Color.Blue);
```
![grafik](https://user-images.githubusercontent.com/29866610/178736962-063c33b9-eff1-4633-b6e4-a23bcec4a485.png)

If I add a second point, the two points are showing up like expected.
```
WpfPlot1.Plot.AddSignal(new double[] { 1, 2 }, 1, System.Drawing.Color.Blue);
```
![grafik](https://user-images.githubusercontent.com/29866610/178737004-1d4542c2-df9d-4704-a023-619af8b45397.png)

Is this the desired bahaviour? For me it would be nice to plot the single point as well here.

**Comment by bclehmann** [comment]
Yeah, this is a bug, thank you for reporting it. I opened a PR to fix it: #1951

**Comment by Fruchtzwerg94** [comment]
Great, thanks!
